src/app/admin/usersTable/usersTableFilters.ts

```diff
@@ -48,32 +34,23 @@ export const emailSearch = (
     return query.ilike("email", `%${state}%`);
 };
 
-export const buildUserRoleFilter = async (): Promise<BuildUserRoleFilterAndError> => {
+export const buildUserRoleFilter = (): Filter<UserRow, string[]> => {
     const userRoleSearch = (
         query: PostgrestFilterBuilder<Database["public"], any, any>,
         state: string[]
     ): PostgrestFilterBuilder<Database["public"], any, any> => {
         return query.in("role", state);
     };
 
-    const { data, error } = await supabase.from("profiles").select("role");
-
-    if (error) {
-        const logId = await logErrorReturnLogId("Error with fetch: User role filter options", {
-            error: error,
-        });
-        return { data: null, error: { type: "failedToFetchUserRoleFilterOptions", logId: logId } };
-    }
-
-    const userRolesFromDb: UserRole[] = data.map((row) => row.role) ?? [];
+    const allUserRoles: UserRole[] = ["volunteer", "manager", "staff", "admin"];
```

Comment:
Yeah i think typing this as UserRole[] is good enough. It can at least cope with renaming of roles. It just can't cope with adding a role in the database, but it's a compromise i guess